include/ui/memory-component-presenter.hpp

```diff
@@ -98,6 +99,15 @@ class MemoryComponentPresenter : public QAbstractListModel {
    */
   QHash<int, QByteArray> roleNames() const Q_DECL_OVERRIDE;
 
+  /**
+   * returns the number of bytes to be displayed in one memory cell.
+   * the role is an indicator of how many bytes are chosen by the user.
+   *
+   * /param role the display role for this cell in the QAbstractListModel
```

Comment:
`\param` and then the first letter after the variable name should be capitalized (it's the start of a sentence):
`\param role The`